From a5d0e9b6d2858c713bd733104330181f95d6c3d8 Mon Sep 17 00:00:00 2001
From: Jesse Barnes <jbarnes@virtuousgeek.org>
Date: Mon, 22 Sep 2008 13:26:58 -0700
Subject: [PATCH 15/20] Add no TV out quirk for HP Compaq nx6110

Hopefully we can fix the VBT parsing code so we don't need this, but for the
time being it seems necessary.  Fixes bz #17683.
(cherry picked from commit 6a46022df57d09109fc119fedce99c156715a4d4)
---
 src/i830_quirks.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git xf86-video-intel-2.4.3/src/i830_quirks.c xf86-video-intel-2.4.3/src/i830_quirks.c
index eb8bd82..785f32e 100644
--- xf86-video-intel-2.4.3/src/i830_quirks.c
+++ xf86-video-intel-2.4.3/src/i830_quirks.c
@@ -310,6 +310,8 @@ static i830_quirk i830_quirk_list[] = {
     /* Samsung Q45 has no TV output */
     { PCI_CHIP_I965_GM, 0x144d, 0xc510, quirk_ignore_tv },
 
+    /* HP Compaq nx6110 has no TV output */
+    { PCI_CHIP_I915_GM, 0x103c, 0x099c, quirk_ignore_tv },
     /* HP Compaq 6730s has no TV output */
     { PCI_CHIP_GM45_GM, 0x103c, 0x30e8, quirk_ignore_tv },
     /* HP Pavilion ze4944ea needs pipe A force quirk (See LP: #242389) */
-- 
1.6.2

